CONTRIBUTING.md

```diff
@@ -1,109 +1,93 @@
-</PRE>
-
-The "tr -d '\015'" is needed with Cygwin to strip carriage returns and has no
-effect in Linux.
-
-The `discouraged` file can also be regenerated with mmj2, which currently runs
-faster than metamath.exe for this function.
-See the Travis script for the details.
-
-
-The metamath.exe and mmj2 proof assistants will prevent most accidental
-violations.  The behavior of the metamath.exe proof assistant in the presence
-of these tags and how to override them is described in the 11-May-2016 entry of
-http://us.metamath.org/mpeuni/mmnotes.txt.
-
-Please note that when you regenerate the `discouraged` file, before comitting
-it you should compare it to the existing one to make sure the differences are
-what you expected and there are no accidental changes elsewhere.
-The purpose of this file is to help detect such accidental changes, and if it
-is not inspected manually that purpose is defeated.
+# Contributing and Acceptance
+
+We're a friendly community, and we would love to have more collaborators!
+
+This document briefly explains
+[how to contribute proposed changes](#how-can-i-contribute)
+("contributions" or "merge requests") to this collection of databases
+It also describes
+[how these proposed changes are evaluated](#what-are-the-requirements-for-acceptance-merging)
+to decide if they will be accepted, since presumably you'll want to
+meet those criteria when creating a contribution.
+
+## How can I contribute?
+
+First, take a look at some existing proofs.
+Walking through the 
+[Metamath Proof Explorer (MPE, aka set.mm)](http://us.metamath.org/mpegif/mmset.html) is a good way to start.
+
+To create a proof, you need to pick a tool.
+The Metamath system specifies a common file format, not a single tool
+everyone has to use.
+Many in the community use the [mmj2 tool](http://us.metamath.org#book).
+The video [Introduction to Metamath and mmj2](https://www.youtube.com/watch?v=Rst2hZpWUbU) demonstrates using mmj2.
+The mmj2 tool includes an interactive tutorial, and if you just want to watch,
+there's a video giving a
+[Walkthrough of the tutorial in mmj2](https://www.youtube.com/watch?v=87mnU1ckbI0).
+You should start small and try to reprove what's already
+proven with Metamath before you try to prove something new.
+If you're contributing to set.mm or iset.mm, take a look at its
+[conventions and style](http://us.metamath.org/mpegif/conventions.html).
+
+At some point you should join the
+[Metamath mailing list](https://groups.google.com/g/metamath).
+
+The Wiki page
+[Getting started with contributing](https://github.com/metamath/set.mm/wiki/Getting-started-with-contributing) will walk you the process of contributing
+a final proof once you have one.
+Basically, you'll create a "pull request" on GitHub to request
+merging of some change(s).
+**Warning**: Changes are made to the **develop** branch.
+
```

Comment:
I would change the order of these sentences: First say where to contribute (GitHub), and then how. 
Suggestion: 
"For contributing a final proof once you have one, you'll basically create a "pull request" on GitHub to request
merging of some change(s) into the corresponding Git repository.
**Warning**: Changes are made to the **develop** branch.

The Wiki page [Getting started with contributing](https://github.com/metamath/set.mm/wiki/Getting-started-with-contributing) will walk you the process of setting up the GitHub environment and contributing your proof."

Reply:
That makes sense! Done.

Reply:
Done.